Extensions/ValuesOfMultipleObjects.json

```diff
@@ -0,0 +1,1796 @@
+      "objectGroups": []
+    },
+    {
+      "description": "Minimum Y position of picked object instances (using AABB of objects)",
+      "fullName": "Minimum Y position of picked object instances (using AABB of objects)",
+      "functionType": "Expression",
+      "name": "MinY_AABB",
+      "private": false,
+      "sentence": "",
+      "events": [
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": "// Get the first instance picked\nconst object = objects[0];\n\n// If the instance doesn't exist we exit this function, returnValue will be 0\nif (!object) return;\n\n// Get the AABB of the object\nconst aabb = object.getAABB();\n\n// Get the layer of the current object\nconst layer = runtimeScene.getLayer(object.getLayer());\n\neventsFunctionContext.returnValue = aabb.min[1];\n",
+          "parameterObjects": "objects",
+          "useStrict": true,
+          "eventsSheetExpanded": true
+        }
+      ],
+      "parameters": [
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "objects",
+          "longDescription": "",
+          "name": "objects",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "objectList"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "Maximum Y position of picked object instances (using AABB of objects)",
+      "fullName": "Maximum Y position of picked object instances (using AABB of objects)",
+      "functionType": "Expression",
+      "name": "MaxY_AABB",
+      "private": false,
+      "sentence": "",
+      "events": [
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": "// Get the first instance picked\nconst object = objects[0];\n\n// If the instance doesn't exist we exit this function, returnValue will be 0\nif (!object) return;\n\n// Get the AABB of the object\nconst aabb = object.getAABB();\n\n// Get the layer of the current object\nconst layer = runtimeScene.getLayer(object.getLayer());\n\neventsFunctionContext.returnValue = aabb.max[1];\n",
+          "parameterObjects": "objects",
+          "useStrict": true,
+          "eventsSheetExpanded": true
+        }
+      ],
+      "parameters": [
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "objects",
+          "longDescription": "",
+          "name": "objects",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "objectList"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "Minimum X position of picked object instances (using center of objects)",
+      "fullName": "Minimum X position of picked object instances (using center of objects)",
+      "functionType": "Expression",
+      "name": "MinX_Center",
```

Comment:
I'm not sure I see the point in duplicating all Min/Max X/Y expressions duplicated for the AABB like that. I think most of the time if people will want the Min and Max X/Y, they want a bounding box around all objects, where the center one would "cut out" a part of the outer bounding objects, and I don't see that being of any use to anyone, while complexifying the extension (as a user, you don't want to have multiple things doing seemingly the same thing, or to have to care about the different implementation details). I'd just leave the AABB ones and remove the `_AABB` parts of their names.

Reply:
In the beginning, I thought the centers might be interesting for some developers.  But now I agree with you that most people will just want the outsides of the objects, which will work for both camera movement and future flocking behaviors.

I have removed the min/max functions based on center points and renamed the AABB versions so they are more user-friendly like you suggested.